src/classes/VOL_CTRL_VolunteersReportHours.cls

```diff
@@ -264,6 +274,13 @@ global virtual with sharing class VOL_CTRL_VolunteersReportHours {
         		listHours[0].Start_Date__c = vhours.Start_Date__c;
         		listHours[0].End_Date__c = vhours.End_Date__c;
         		listHours[0].Hours_Worked__c = vhours.Hours_Worked__c;
+        	
+        		// update the additional fields, if any
+        		for (Schema.FieldSetMember f : SObjectType.Volunteer_Hours__c.FieldSets.VolunteersReportHoursFS.getFields()) {
+        			String fieldName = f.getFieldPath();
+		        	listHours[0].put(fieldName, vhours.get(fieldName));		
+		        }	
+        		   		
```

Comment:
I check now if the new value is null before updating. I don't think we need to do anything on the insert - the vhours contains the comments, as they are set on the page, just like the hours.
(I did forget to include the page in my previous commit - sorry about that. I added the page now.)